tests/inputData.json.example

```diff
@@ -57,6 +64,13 @@
 		"NSX Edge Nodes": [
 			"sfo-w01-en01",
 			"sfo-w01-en02"
+		],
+		"NSX Edge Fqdn": [
+			"sfo-w01-en01.sfo.rainpole.io",
+			"sfo-w01-en02.sfo.rainpole.io"
+		],
```

Comment:
Should we replace NSX Edge Nodes which just have the hostnames and use the FQDN in there instead?

Reply:
We can look at it later, because this requires some code change. Like just having hostname and then read domain name from SDDC manager and so on. Instead I think I have a ticket to enhance pester to read everything from SDDC manager if possible than from JSON. We can take it up there if you don't mind.